### ISSUE TITLE
Code structure

### ISSUE THREAD

**Issue by ikennamax** [classEze/max-assessment#4]
All the code happens to in one file `server.js`, without any logical/functional separation into files for cleanness and modularity

**Comment by classEze** [comment]
Yes. This is intentional. I reasoned this is a small project and bringing in some of the above mentioned e.g modularity and maybe a router(express router) for separation of route groups would be overkill.

**Comment by ikennamax** [comment]
Hmm. Well, fair enough, I do understand. But it depends on how you see it. Even if the project is "small" (which is subjective), it does not mean that you can't have proper logical code separation. That affects overall readability, and anyone who is going to look at your code will appreciate that they don't have to deal with everything lumped together in the same file. 

We want to get the feeling that you "care" enough about the code solution (not just wanting to slap code blocks together 😄), and many times that "care" shows up in the way the code is presented. 

Don't get me wrong - if the code had run successfully and we were able to test it and the tests pass, this code structure issue will likely not be a show stopper for you overall.

I hope I am communicating something valuable here?

**Comment by classEze** [comment]
Yes. Thanks for the feedback.
